### DISCUSSION TITLE
Q: REI recipes not working properly since update

### DISCUSSION THREAD

**Discussion by Soivet** [shedaniel/RoughlyEnoughItems#878] [Q & A] [answered]
I am hosting AOF5 on my server and updated from 1.2.0 of their pack to 1.2.3
Since that upgrade, left clicking on a recipe works to get it, but then left clicking on the components doesn't show a recipe.
Checked on server and client. It seems to work with keybind R but now with the left click.
Fresh install or not doesnt change anything. Got any clues to why this might be?
Maybe a fabric loader out of date?

**Comment by shedaniel** [comment, marked as answer]
REI has an update to fix this! Please update!
